target/snitch_cluster/README.md

```diff
@@ -29,19 +65,25 @@ questa-2022.3 make bin/snitch_cluster.vsim
 vcs-2020.12 make bin/snitch_cluster.vcs
 ```
 
+You can find more details of the configuration file in the Cluster Configuration below.
+
 These commands compile the RTL sources respectively in `work-vlt`, `work-vsim` and `work-vcs`. Additionally, common C++ testbench sources (e.g. the [frontend server (fesvr)](https://github.com/riscv-software-src/riscv-isa-sim)) are compiled under `work`. Each command will also generate a script or an executable (e.g. `bin/snitch_cluster.vsim`) which you can invoke to simulate the hardware. We will see how to do this in a later section.
 
 ### Building the Banshee simulator
 Instead of running an RTL simulation, you can use our instruction-accurate simulator called `banshee`. To install the simulator, please follow the instructions of the Banshee repository: [https://github.com/pulp-platform/banshee](https://github.com/pulp-platform/banshee).
 
-### Cluster configuration
+### Cluster Configuration
 
 Note that the Snitch cluster RTL sources are partly automatically generated from a configuration file provided in `.hjson` format. Several RTL files are templated and use the `.hjson` configuration file to fill the template entries. An example is `/hw/snitch_cluster/src/snitch_cluster_wrapper.sv.tpl`.
 
 Under the `cfg` folder, different configurations are provided. The `cfg/default.hjson` configuration instantiates 8 compute cores + 1 DMA core in the cluster. If you need a specific configuration you can create your own configuration file.
 
-The command you executed previously automatically generated the templated RTL sources. It implicitly used the default configuration file.
-To override the default configuration file, define the following variable when you invoke `make`:
+We have other architectures for different accelerators:
+* `cfg/snax-mac.hjson` - is a SNAX shell with the simple [HWPE MAC engine](https://github.com/pulp-platform/hwpe-mac-engine).
```

Comment:
Are we still using our fork here? Shouldn't we point to our fork instead then?

Reply:
Ah yeah okay, will put our own fork here 😄